### PR TITLE
fix(login-flow): tooltips for external link account

### PR DESCRIPTION
**Pull request by kutsko-applicaster**
## Description

[Monday ticket](https://applicaster.monday.com/boards/1615228456/pulses/3912817258)

Added tooltips to describe the external link account entitlement

<details>
  <summary>Platforms Affected</summary>
  
- [ ] Android
- [ ] Android TV
- [ ] Apple TV (TVOS)
- [ ] Fire TV (FireOS)
- [ ] iOS
- [ ] LG TV (WebOS)
- [ ] Samsung TV (Tizen)
- [ ] Web
</details>

## Known or Pending Issues

`fill in this section with potential issues / limitations the reviewer(s) should know about`

## Screenshots

`If there is some kind of ui change please share screenshots`

## Version Info

`Does this update require a certain version of quickbrick, or does it rely on another pr to work?`

### CODE REVIEW

File: plugins/quick-brick-login-flow/manifests/manifest.config.js [hunk 1]
```diff
@@ -551,8 +552,9 @@ const custom_configuration_fields = {
   apple: custom_configuration_fields_default.concat([
       {
         group: true,
-        label: "External Registration Link",
+        label: "External Registration Link (iOS only)",
         label_tooltip: "Use external link account",
+        tooltip: "The external link account entitlement (Apple doc - https://developer.apple.com/support/reader-apps/). This entitlement is used to present intermediate screen before moving to external link",
```
Comment (ranmeirman) [inline plugins/quick-brick-login-flow/manifests/manifest.config.js:557]: Please update the tooltip:
"The External Link Account Entitlement, as detailed in Apple's documentation (https://developer.apple.com/support/reader-apps/), enables the display of an intermediary screen before navigating to an external link. To utilize this feature, prior approval from Apple is mandatory."

File: plugins/quick-brick-login-flow/manifests/manifest.config.js [hunk 2]
```diff
@@ -188,6 +188,7 @@ const baseManifest = {
         group: true,
         label: "External Registration Link (iOS only)",
         label_tooltip: "Use external link to register an account",
+        tooltip: "The external link account entitlement (Apple doc - https://developer.apple.com/support/reader-apps/). This entitlement is used to present intermediate screen before moving to external link",
```
Comment (ranmeirman) [inline plugins/quick-brick-login-flow/manifests/manifest.config.js:191]: Please update the tooltip:
"The External Link Account Entitlement, as detailed in Apple's documentation (https://developer.apple.com/support/reader-apps/), enables the display of an intermediary screen before navigating to an external link. To utilize this feature, prior approval from Apple is mandatory."

**Review by ranmeirman — CHANGES_REQUESTED** [review]
Please update and merge

File: plugins/quick-brick-login-flow/manifests/manifest.config.js (hunk 1 above)
Comment (kutsko-applicaster) [reply plugins/quick-brick-login-flow/manifests/manifest.config.js]: done

File: plugins/quick-brick-login-flow/manifests/manifest.config.js (hunk 2 above)
Comment (kutsko-applicaster) [reply plugins/quick-brick-login-flow/manifests/manifest.config.js]: done

**Review by ranmeirman — APPROVED** [review]

**Pull request merged**